l1-contracts/src/core/Rollup.sol

```diff
@@ -900,19 +905,17 @@ contract Rollup is Leonidas, IRollup, ITestRollup {
       revert Errors.Rollup__InvalidArchive(tipArchive, _header.lastArchive.root);
     }
 
-    uint256 slot = _header.globalVariables.slotNumber;
-    if (slot > type(uint128).max) {
```

Comment:
This check is lost deliberately?

Reply:
Yep. The check was there because we at some point were storing just a `uint128` for storage packing, but hence half of it is not finished yet or in flux, seemed better to just insert the `Slot` type now, and then use the full size until we actually need to handle try and pack it.